proposals/lip-0054.md

```diff
@@ -107,7 +107,7 @@ Furthermore, for the rest of this section:
 
 * Let `RMTVerify` be the `verifyDataBlock` function specified in [the appendix C of LIP 0031][lip-0031#appendixC].
 * Let `RMTCalculateRoot` be the `calculateRootFromUpdateData` function specified in [the appendix E of LIP 0031][lip-0031#appendixE].
-* Let `SMTVerify` be the `verify` function specified in [LIP 0039][lip-0039].
+* Let `SMTVerifyInclusionProof` be the `verify_inclusion_proof` function specified in [LIP 0039][lip-0039].
```

Comment:
There is no specified function in `lip-0039` with name verify_inclusion_proof, I guess we should change the name of the `verify` function on that file too.

Reply:
It is specified in this PR: https://github.com/LiskHQ/lips/pull/431